airflow/hooks/dbapi_hook.py

```diff
@@ -150,12 +150,19 @@ def run(self, sql, autocommit=False, parameters=None):
         :param autocommit: What to set the connection's autocommit setting to
             before executing the query.
         :type autocommit: bool
+        :param run_as_script: Split sql into individual statements
+        :type run_as_script: bool
         :param parameters: The parameters to render the SQL query with.
         :type parameters: mapping or iterable
         """
         conn = self.get_conn()
         if isinstance(sql, basestring):
-            sql = [sql]
+            if run_as_script:
+                sql = sql.split(';')    # split statements using ';'
```

Comment:
This is brittle, any statement that would contain a `;` within a string would fail.
```
SELECT 'fail;' FROM tbl;
```
Maybe a regexp that disregards the content of quoted strings. Maybe `sqlparse` is able to do this nicely.

Reply:
Good point. I will fix this to use regexp and will also check `sqlparse`. Thanks @mistercrunch